api/edge_api/identities/export.py

```diff
@@ -0,0 +1,181 @@
+                        featurestate_uuid,
+                        feature_id,
+                        override["enabled"],
+                    )
+                )
+                featurestate_value = override["feature_state_value"]
+                if featurestate_value is not None:
+                    # export feature state value
+                    identity_override_export.append(
+                        export_featurestate_value(featurestate_value, featurestate_uuid)
+                    )
+                if mvfsv := override["multivariate_feature_state_values"]:
+                    # only one mvfs override can exists
+                    mv_feature_option_id = mvfsv[0]["multivariate_feature_option"]["id"]
+                    # export mv feature state value
+                    identity_override_export.append(
+                        export_mv_featurestate_value(
+                            featurestate_uuid, mv_feature_option_id
+                        )
+                    )
+        if "LastEvaluatedKey" not in response:
+            break
+        kwargs["start_key"] = response["LastEvaluatedKey"]
+    return identity_export, traits_export, identity_override_export
+
+
+def _get_feature_uuid(feature_id: int) -> str:
+    return Feature.objects.get(id=feature_id).uuid
+
+
+def _get_mv_feature_option_uuid(mv_feature_option_id: int) -> str:
+    return MultivariateFeatureOption.objects.get(id=mv_feature_option_id).uuid
+
+
+def export_edge_trait(trait: dict, identifier: str, environment_api_key: str) -> dict:
+    trait_value = map_any_value_to_trait_value(trait["trait_value"])
+    trait_value_data = Trait.generate_trait_value_data(trait_value)
+    return {
+        "model": "traits.trait",
+        "fields": {
+            "identity": [identifier, environment_api_key],
+            "created_date": timezone.now().isoformat(),
+            "trait_key": trait["trait_key"],
+            **trait_value_data,
+        },
+    }
+
+
+def export_edge_identity(
+    identifier: str, environment_api_key: str, created_date: str
+) -> dict:
+    return {
+        "model": "identities.identity",
+        "fields": {
+            "identifier": identifier,
+            "created_date": created_date,
+            "environment": [environment_api_key],
+        },
+    }
+
+
+def export_edge_feature_state(
+    identifier: str,
+    environment_api_key: str,
+    featurestate_uuid: str,
+    feature_id: int,
+    enabled: bool,
+) -> dict:
+    global feature_id_to_uuid
+    if feature_id not in feature_id_to_uuid:
+        feature_id_to_uuid[feature_id] = _get_feature_uuid(feature_id)
+    feature_uuid = feature_id_to_uuid[feature_id]
+
+    return {
+        "model": "features.featurestate",
+        "fields": {
+            "uuid": featurestate_uuid,
+            "created_at": timezone.now().isoformat(),
+            "updated_at": timezone.now().isoformat(),
+            "live_from": timezone.now().isoformat(),
```

Comment:
It's a little surprising that these timestamps are all set to `now()` maybe add a comment explaining why?

Reply:
Yeah, not a bad idea